app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,32 @@
+class FoodsController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    @foods = current_user.foods
+  end
+
+  def new
+    @food = Food.new
+  end
+
+  def create
+    @food = current_user.foods.build(food_params)
+    if @food.save
+      redirect_to foods_path, notice: 'Food created successfully'
+    else
+      render :new
+    end
+  end
+
+  def destroy
+    @food = Food.find(params[:id])
+    @food.destroy
+    redirect_to foods_path
+  end
```

Comment:
- [ ] @team great work adding a method to remove food, kindly consider using best practice by providing the path for success and failure of the deletion as you did for the create method.

Reply:
**Thank you for suggesting a great point😎**
 we will add it in the next commit!